README.md

```diff
@@ -124,4 +124,15 @@ LOG_LEVEL=INFO
 You can connect your logs to Loogly, request a token at Loggly.com and add the token to the variable LOGGLY_TOKEN
 ```bash
 LOGGLY_TOKEN=<your token here>
-```
\ No newline at end of file
+```
+## Kubectl
+To run this app on k8s you will have to create the secrets. The easiest way is:
+```bash
+kubectl create secret generic myapp-secrets --from-literal=SECRET_KEY=<secret> --from-literal=GITHUB_CLIENTSECRET=<secret> --from-literal=LOGGLY_TOKEN=<secret>--from-literal=GITHUB_CLIENTID=<secret> --from-literal=MONGO_CONNECTION_STRING=<connection>
+```
+Don't forget to forward the port:
+```bash
+kubectl port-forward service/module-14 7080:80
```

Comment:
Would choosing port 5000 make more sense, in order to allow your existing OAuth flow to hit the right URL? Otherwise you get redirected to localhost:5000 if you're using the same Client Id/secret as you use in non-kubernetes development